### PR TITLE
Add description of available timeseries forecasting models

### PR DESCRIPTION
**Pull request by shchur**
*Description of changes:*
- Fix several problems with hyperlinks in forecasting tutorials
- Add a page describing available forecasting models & their hyperparameters
- Update docstrings for all models to include correct hyperparameters

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/tutorials/timeseries/forecasting-indepth.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This tutorial provides an in-depth overview of the time series forecasting capab
     - Hyperparameter tuning
     - Forecasting irregularly-sampled time series
 
-This tutorial assumes that you are familiar with the contents of the [basic tutorial](forecasting-quickstart.md).
+This tutorial assumes that you are familiar with the contents of the [basic tutorial](forecasting-quickstart.html).
```
Comment (gradientsky) [inline docs/tutorials/timeseries/forecasting-indepth.md:15]: I think idiomatic way to reference other pages is to have a label in the target doc [see source](https://github.com/awslabs/autogluon/blob/master/docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-training.md?plain=1#L2):
```
:label:`cloud_aws_sagemaker_fit`
```
And then you can [do a reference](https://github.com/awslabs/autogluon/blob/master/docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md?plain=1#L4):
```
:ref:`cloud_aws_sagemaker_fit`
```

File: docs/tutorials/timeseries/forecasting-indepth.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ Finally, an **ensemble** model works by combining predictions of all other model
 By default, `TimeSeriesPredictor` always fits a `WeightedEnsemble` on top of other models.
 This can be disabled by setting `enable_ensemble=False` when calling the `fit` method.
 
-For a list of tunable hyperparameters for each model, their default values, and other details see [Model zoo](#TODO).
+For a list of tunable hyperparameters for each model, their default values, and other details see [Forecasting Time Series - Model Zoo](forecasting-model-zoo.html).
```
Comment (gradientsky) [inline docs/tutorials/timeseries/forecasting-indepth.md:82]: ditto

File: docs/tutorials/timeseries/forecasting-indepth.md [hunk 3]
```diff
@@ -216,7 +217,7 @@ The code above will only train two models:
 - `DeepAR` (with default hyperparameters)
 - `ETS` (with the given `seasonality` and `seasonal_period`; all other parameters set to their defaults).
 
-For the full list of available models and the respective hyperparameters, see [Model zoo](#TODO).
```
Comment (gradientsky) [inline docs/tutorials/timeseries/forecasting-indepth.md:219]: links

File: docs/tutorials/timeseries/forecasting-quickstart.md [hunk 4]
```diff
@@ -255,4 +255,4 @@ predictor.fit(
 # Generate the forecasts
 predictions = predictor.predict(ts_dataframe)
 ```
-Check out the [in-depth tutorial](forecasting-indepth.md) to learn about the advanced capabilities of AutoGluon for time series forecasting.
+Check out the [in-depth tutorial](forecasting-indepth.html) to learn about the advanced capabilities of AutoGluon for time series forecasting.
```
Comment (gradientsky) [inline docs/tutorials/timeseries/forecasting-quickstart.md:258]: links

File: docs/tutorials/timeseries/forecasting-model-zoo.rst [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+
+
+Models
+------
+
+This page contains the list of time series forecasting models available in AutoGluon.
+The available hyperparameters for each model are listed under **Other parameters**.
+
+This list is useful if you want to `override the default hyperparameters <forecasting-indepth.html#manually-configuring-models>`_ or `define custom hyperparameter search spaces <forecasting-indepth.html#hyperparameter-tuning>`_, as described in the In-depth Tutorial.
```
Comment (gradientsky) [inline docs/tutorials/timeseries/forecasting-model-zoo.rst:18]: links

File: docs/tutorials/timeseries/forecasting-indepth.md (hunk 1 above)
Comment (shchur) [reply docs/tutorials/timeseries/forecasting-indepth.md:15]: Thanks for the pointer!

**Review by huibinshen — APPROVED** [review]
